packages/hierarchies-react/package.json

```diff
@@ -64,24 +64,19 @@
     "rxjs": "catalog:rxjs"
   },
   "peerDependencies": {
-    "@itwin/itwinui-icons": "5.0.0-alpha.7",
-    "@itwin/itwinui-react": "5.0.0-alpha.14",
+    "@stratakit/icons": "~0.1.0",
```

Comment:
@mayank99 Should libraries built on top of stratakit specify `@stratakit/icons` as peer dep or only `@stratakit/foundations` should be peer dep if used directly by the library?

Reply:
If you are good with it, consider making all `@stratakit` packages peer deps. It makes some teams uncomfortable but is really the best approach in the end.

From our side, only `@stratakit/foundations` is required to be a peer dep. But I can see the argument for `@stratakit/icons` too, as it requires [bundler configuration](https://github.com/iTwin/design-system/tree/main/packages/icons#bundler-configuration) (something that packages cannot do).

Reply:
Agreed about documentation. We need to document the requirements clearly. In addition to direct vs peer deps, there is also a question of `^` vs `~` which needs to be documented.

(Short answer: you should use `^`, because `~` will exclude patch versions when pre-1.0. Breaking changes will only be published in minor versions, which are automatically excluded when using `^`, so it's safe.)

Reply:
> Short answer: you should use `^`, because `~` will exclude patch versions when pre-1.0. Breaking changes will only be published in minor versions, which are automatically excluded when using `^`, so it's safe.

That's all semver stuff, so not sure it needs additional docs besides highlighting the fact that your packages follow semver.